docs/Components/Aerodynamic/RectangularHorizontalStabilizer.md

```diff
@@ -0,0 +1,24 @@
+## RectangularHorizontalStabilizer
+This class is responsible for creating a rectangular horizontal stabilizer type component. For this, it is necessary to pass as parameters the position and the angle in relation to the parent component, as well as the chord and span of the rectangular horizontal stabilizer.
+```python
+from vec import Vector2
+import math
+import pytest
+from adr.Components.Aerodynamic import RectangularHorizontalStabilizer
+
+rectangular_horizontal_stabilizer = RectangularHorizontalStabilizer(
+    relative_position=Vector2(x=-0.7, y=0.2),
+    relative_angle=math.radians(0),
+    span=0.15,
+    chord=0.20
+)
```

Comment:
There are two parameters missing, both on the instructions and instantiation: name and mass.

Despite that, the description is much more intuitive!